portfolio_style.css

```diff
@@ -0,0 +1,304 @@
+* {
+	box-sizing: border-box;
+}
+
+html {
+	scroll-behavior: smooth;
+	font-family: "Calibre","Inter","San Francisco","SF Pro Text",-apple-system,system-ui,sans-serif;
+	font-size: 10px;
+	}
+
+body {
+	background-color: #01274a;
+	color: #c0c2c2;
+}
+
+main {
+	padding: 0px 150px;
+}
+
+.nav_bar {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-end;
+	flex-wrap:nowrap;
+	gap: 10px;
+	padding: 30px;
+	align-items: stretch;
+	width: 100%;
+	position: absolute;
+	top: 0px;
+	left: 0px;
+	background-color: #01274a;
+	
+}
+
+.nav_button {
+	color: #1fffe1;
+	text-decoration: none;
+	font-size: 2rem;
+	font-family: 'Roboto Mono', monospace;
+	font-weight: 100;
+}
+
+.nav_button_text {
+	color: #c0c2c2;
+}
+
+.nav_button_text:hover {
+	color: #1fffe1;
+	transition: 0.5s;
+}
+
+.splash_container{
+	height: 100vh;
+}
+
+.intro_container{
+	display: flex;
+	flex-wrap: wrap;
+	flex-direction: column;
+	align-items: flex-start;
+	justify-content: center;
+	height: 100vh;
+
+}
+
+.initial_hi{
+	color: #1fffe1;
+	font-family: 'Roboto Mono', monospace;
+	font-size: 2rem;
+	font-weight: 400;
+}
+
+h1 {
+	font-size: 6rem;
+	margin: 10px 0px;
+	color: #b0cfcf;
+}
+
+.intro_subheading {
+	font-size: 5rem;
+	margin: 10px 0px;
+	color: #87a1a1;
+}
+
+.intro_short {
+	font-size: 2rem;
+	color: #6e8585;
+}
+
+.intro_button, .git_button {
+	background-color: #01274a;
+	text-decoration: none;
+	color: #1fffe1;
+	border: #1fffe1 solid 1px;
+	font-family: 'Roboto Mono', monospace;
+	font-size: 2rem;
+	padding: 20px;
+	border-radius: 10px;
+}
+
+.intro_button:hover, .git_button:hover {
+	background-color: #1fffe1;
+	color: #01274a;
+}
+
+.down_arrow {
```

Comment:
this thing doesn't work on firefox because it's really difficult to click (as discussed)